packages/landing/src/app/style.css

```diff
@@ -0,0 +1,126 @@
+@font-face {
+  font-family: "PlusJakartaSans";
+  font-display: swap;
+  src: url("/fonts/PlusJakartaSans-VariableFont_wght.ttf") format("truetype");
```

Comment:
Could you please use `next/font` as in the previous PR?

Reply:
Ok, fixed